### DISCUSSION TITLE
Change syntax to be more C-like, not C++-like, while preserving backward compatibility and memory-safe features

### DISCUSSION THREAD

**Discussion by ghost** [crablang/crab#67] [Ideas]
Currently, Rust syntax resembles C++ syntax, which is not good for beginners. Thus, it'll be better when there will be less clutter in this language since I currently code with V.

**Comment by trvswgnr** [comment]
there's been some discussion around a language with a simpler syntax for beginners, and i think that would be fun, but it's not in the scope of this project.
